pinc/DifferenceEngineWrapperBB.inc

```diff
@@ -0,0 +1,24 @@
+<?php
+// This file inherits from DifferenceEngineWrapper to facilitate
+// formatting diffs using BB code.
+include_once($relPath."DifferenceEngineWrapper.inc");
+include_once($relPath."BBDiffFormatter.inc");
+
+class DifferenceEngineWrapperBB extends DifferenceEngineWrapper
```

Comment:
This seems unsymmetrical somehow. Alternatively you could make DifferenceEngineWrapper into an abstract class and extend DifferenceEngineWrapperBB and DifferenceEngineWrapperTable from it. They would have their own constructors and definitions of addHeader() and everything else derived from the parent.

Reply:
Done - it also led me to a way of easily providing the suggested Copy to Clipboard button. Thanks